introduction.md

```diff
@@ -1,2 +1,8 @@
 I'm Jolyan, aka Jools!
 Currently liking the Electives.
+
+Deadly Skills
+*PHP
+*JS
+*GitHub :)
+
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?